canvas.go

```diff
@@ -385,7 +402,7 @@ func (self *Canvas) Thumbnail(width uint, height uint) error {
 
 // Puts a canvas on top of the current one.
 func (self *Canvas) AppendCanvas(source *Canvas, x int, y int) error {
-	success := C.MagickCompositeImage(self.wand, source.wand, C.OverCompositeOp, C.ssize_t(x), C.ssize_t(y))
+	success := C.MagickCompositeImage(self.wand, source.wand, C.SrcOverCompositeOp, C.ssize_t(x), C.ssize_t(y))
```

Comment:
I think it's more complicated than that because from what I understand, they just have a different output. So I wouldn't change that in such a pull request.